<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="wallpaper">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>480</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/pantallaPrincipal/Fondo.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="divisor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>800</width>
      <height>4</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/pantallaPrincipal/divisor.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="time">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>444</y>
      <width>91</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>10:25</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="date">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>444</y>
      <width>200</width>
      <height>36</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font-size: 16px;</string>
    </property>
    <property name="text">
     <string>Jueves 23 de enero</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="tempContainer" native="true">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>310</y>
      <width>770</width>
      <height>110</height>
     </rect>
    </property>
    <widget class="QLabel" name="tempWallpaper">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>770</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/pantallaPrincipal/Fondo Barra Abajo.png</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="timeOn" native="true">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>35</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QLabel" name="timeOnLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8B8B8B;
font-size: 12px;</string>
      </property>
      <property name="text">
       <string>Horas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="timeOnVal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>13</y>
        <width>70</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>9:30</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="trip" native="true">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>35</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QLabel" name="tripLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8B8B8B;
font-size: 12px;</string>
      </property>
      <property name="text">
       <string>Trip</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="tripVal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>13</y>
        <width>70</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>10:01</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="autonomy" native="true">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>35</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <widget class="QLabel" name="autonomyLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>10</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8B8B8B;
font-size: 12px;</string>
      </property>
      <property name="text">
       <string>Autonom√≠a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="autonomyVal">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>13</y>
        <width>70</width>
        <height>24</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>10:01</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="inversorTemp" native="true">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>24</y>
       <width>110</width>
       <height>62</height>
      </rect>
     </property>
     <widget class="QLabel" name="inversorTempIcon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>28</width>
        <height>62</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/pantallaPrincipal/temp_icono_vacia.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="inversorLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>6</y>
        <width>90</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8B8B8B;
font-size: 10px;</string>
      </property>
      <property name="text">
       <string>Inversor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="inversorValue">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>22</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="engineTemp" native="true">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>24</y>
       <width>110</width>
       <height>62</height>
      </rect>
     </property>
     <widget class="QLabel" name="engineTempIcon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>28</width>
        <height>62</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/pantallaPrincipal/temp_icono_vacia.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="engineTempLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>6</y>
        <width>90</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8B8B8B;
font-size: 10px;</string>
      </property>
      <property name="text">
       <string>Motor</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="engineTempValue">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>22</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="batTemp" native="true">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>24</y>
       <width>110</width>
       <height>62</height>
      </rect>
     </property>
     <widget class="QLabel" name="batTempIcon">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>28</width>
        <height>62</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/pantallaPrincipal/temp_icono_vacia.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="batTempLabel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>6</y>
        <width>90</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #8B8B8B;
font-size: 10px;</string>
      </property>
      <property name="text">
       <string>Bater√≠a</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="batTempValue">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>22</y>
        <width>80</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QSlider" name="horizontalSlider_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_3">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_4">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>90</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="speedometer" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>-5</y>
      <width>359</width>
      <height>359</height>
     </rect>
    </property>
    <widget class="QLabel" name="speedometerImage">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>359</width>
       <height>359</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/pantallaPrincipal/velocimetro.png</pixmap>
     </property>
    </widget>
    <widget class="QGraphicsView" name="needleContainer">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>16</y>
       <width>327</width>
       <height>327</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>200</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>120</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="speedValue">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>260</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white; font-size: 28px; font-weight: 600</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="speedLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>290</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: white;</string>
     </property>
     <property name="text">
      <string>KM/W</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="batInfo" native="true">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>170</y>
      <width>390</width>
      <height>110</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>110</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/pantallaPrincipal/barra_lateral_bateria.png</pixmap>
     </property>
    </widget>
    <widget class="QWidget" name="voltageHV" native="true">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>0</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="voltageHVLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>8</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 10px;
color: #8B8B8B;</string>
      </property>
      <property name="text">
       <string>Voltaje HV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="voltageHVValue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>22</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px;
color: white;</string>
      </property>
      <property name="text">
       <string>90V</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="voltageLV" native="true">
     <property name="geometry">
      <rect>
       <x>44</x>
       <y>45</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="voltageLVLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 10px;
color: #8B8B8B;</string>
      </property>
      <property name="text">
       <string>Voltaje LV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="voltageLVValue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>27</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px;
color: white;</string>
      </property>
      <property name="text">
       <string>90V</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="currentHV" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>0</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="currentHVLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>8</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 10px;
color: #8B8B8B;</string>
      </property>
      <property name="text">
       <string>Corriente HV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentHVValue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>22</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px;
color: white;</string>
      </property>
      <property name="text">
       <string>90A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="currentLV" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>45</y>
       <width>80</width>
       <height>50</height>
      </rect>
     </property>
     <widget class="QLabel" name="currentLVLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 10px;
color: #8B8B8B;</string>
      </property>
      <property name="text">
       <string>Corriente LV</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="currentLVValue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>27</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 24px;
color: white;</string>
      </property>
      <property name="text">
       <string>90A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="bat" native="true">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>123</y>
      <width>94</width>
      <height>180</height>
     </rect>
    </property>
    <widget class="QLabel" name="batIcon">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>94</width>
       <height>179</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/pantallaPrincipal/bat_icono.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="batFill">
     <property name="geometry">
      <rect>
       <x>7</x>
       <y>13</y>
       <width>80</width>
       <height>159</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string>color: white; font-size: 12px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="batTextContainer" native="true">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>70</y>
       <width>70</width>
       <height>40</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #434343; border-radius: 10px;</string>
     </property>
     <widget class="QLabel" name="batValue">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>70</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; font-size: 24px;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="batValue_2">
      <property name="geometry">
       <rect>
        <x>55</x>
        <y>15</y>
        <width>15</width>
        <height>15</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: white; font-size: 12px;</string>
      </property>
      <property name="text">
       <string>%</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QSlider" name="verticalSlider">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>140</y>
      <width>16</width>
      <height>160</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QWidget" name="notifications" native="true">
    <property name="geometry">
     <rect>
      <x>395</x>
      <y>15</y>
      <width>390</width>
      <height>100</height>
     </rect>
    </property>
    <widget class="QLabel" name="notificationsWallpaper">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>390</width>
       <height>100</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="resources.qrc">:/images/pantallaPrincipal/barra_notificaciones.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>88</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confButton">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>444</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent;
border: none</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="confIcon">
    <property name="geometry">
     <rect>
      <x>764</x>
      <y>444</y>
      <width>36</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="resources.qrc">:/images/pantallaPrincipal/configIcon.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
